tools/run_tests/run_tests.py

```diff
@@ -355,7 +355,7 @@ def test_specs(self):
                             tests = subprocess.check_output(
                                 [binary, '--benchmark_list_tests'],
                                 stderr=fnull)
-                        for line in tests.split('\n'):
+                        for line in tests.decode().split('\n'):
```

Comment:
we need to make sure this still works under python2.7 as well (AFAIK some parts of the infrastructure still use it) - have you tested that?

Reply:
The test runner is still Python 2, so CI will take care of that.